map-card.js

```diff
@@ -27,6 +27,10 @@ class MapCard extends LitElement {
 
   firstUpdated() {
     this.map = this._setupMap();    
+    // force a render when the page is done loading
+    setTimeout(() => {
```

Comment:
I wonder if we could delay calling the firstUpdated() call till page has loaded. Might be a safer way to do it.

Reply:
According to what i read on the documentation, this does not seem like a possibility, closest you can get is by using the `update()` method, but it seems to fire only after user interaction.
I've come up with an arguably sturdier approach using ResizeObserver, i'll open another pull request shortly.